.ci/azure/android_arm64.yml

```
@@ -9,6 +9,8 @@ trigger:
     - /**/docs/*
     - /**/*.md
     - README.md
+    - src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/*
```

Should it be
```suggestion
    - src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/**
```
?
Did you check that these changes work?

---

fixed

---

Did you check that these paths works as expected? :)

---

I made sure :)